go/test/endtoend/recovery/pitr/shardedpitr_test.go

```diff
@@ -399,7 +399,7 @@ func initializeCluster(t *testing.T) (int, error) {
 		for _, tablet := range shard.Vttablets {
 			proc, err := tablet.MysqlctlProcess.StartProcess()
 			if err != nil {
-				return 1, err
+				return err
```

Comment:
I meant, that since you already doing `require.NoError(t, err)` after launchCluster, you don't need to return error from here and then check for noError, rather don't return anything and just do the error check here , so that we can get rid of lot of code . Following can be replace with `require.NoError(t, err)` .

```
if err!=nil {
return err
}
```

Reply:
Changed to require.NoError() function